main.go

```diff
@@ -164,6 +163,14 @@ func libhoneyConfig() libhoney.ClientConfig {
 		APIKey:  apiKey,
 		Dataset: dataset,
 		APIHost: apiHost,
+		Transmission: &transmission.Honeycomb{
+			MaxBatchSize:          libhoney.DefaultMaxBatchSize,
+			BatchTimeout:          libhoney.DefaultBatchTimeout,
+			MaxConcurrentBatches:  libhoney.DefaultMaxConcurrentBatches,
+			PendingWorkCapacity:   libhoney.DefaultPendingWorkCapacity,
+			UserAgentAddition:     fmt.Sprintf("honeycomb-lambda-extension-%s/%s", runtime.GOARCH, version),
```

Comment:
I feel like this could be pulled out for definition, then just used as a variable here... but I don't feel strongly about it, especially if it works as-is here.